lib/services/runtime/handlers/interaction.ts

```diff
@@ -71,15 +71,12 @@ export const InteractionHandler: HandlerFactory<Node.Interaction.Node, typeof ut
     // request for this turn has been processed, delete request
     runtime.turn.delete(T.REQUEST);
 
-    // check for noMatches to handle
-    if (nextId === undefined && utils.noMatchHandler.canHandle(node, runtime)) {
+    // handle noMatch
+    if (nextId === undefined) {
```

Comment:
`noMatchHandler` now handles the `else` case. If no choice + command is matched, use the `noMatch`